Overview.src.html

```diff
@@ -3026,6 +3080,45 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  to `<code title>GET</code>` and <var>request</var>'s <span title=concept-request-body>body</span>
  to null.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+   <span title=concept-header-parse>parsing</span>
+   `<code title>Include-Referred-Token-Binding-ID</code>` in <var>actualResponse</var>'s
+   <span title=concept-response-header-list>header list</span>.
+   The value of this <span title=concept-header>header</span> is specified in
+   <a href="https://tools.ietf.org/id/draft-ietf-tokbind-https-05.xml#rfc.section.3.3">section 3.3</a>
+   of the Token Binding over HTTP spec.
+   <p class="note no-backref">By setting the <code title>Include-Referred-Token-Binding-ID</code>
+   <span title=concept-header>header</span> to "<code title>true</code>",
+   the <span title=concept-request-origin>origin</span> that sends the
+   redirect <span title=concept-response>response</span> tells the user agent to disclose the
+   <span title=concept-token-binding-id>Token Binding Id</span> used by the user agent with that
+   <span title=concept-request-origin>origin</span> to the target origin.
+
+   <li><p>Unset the <var>request</var>'s
+     <span title=concept-request-use-referred-token-binding>use-referred-token-binding</span> flag.
+   <li><p>If <var>useReferredTokenBinding</var> is "<code title>true</code>",
```

Comment:
Result of header value parsing cannot be a string. Perhaps you meant to use ` rather than "? Also, is this a case-sensitive match? Is that how the RFC defines it?